src/adapt/layout.js

```diff
@@ -747,7 +758,7 @@ adapt.layout.Column.prototype.buildViewToNextBlockEdge = function(position, chec
                 if (!adapt.layout.isSpecialNodeContext(position))
                     checkPoints.push(position.copy());
             }
-            self.layoutContext.nextInTree(position).then(function(positionParam) {
+            var cont = self.nextInTree(position).then(function(positionParam) {
```

Comment:
Fix it in [this commit](https://github.com/vivliostyle/vivliostyle.js/pull/348/commits/59c5adcc7392ee273b8220fa13e970e90c756b4e).